src/code/code_80097A00.c

```diff
@@ -78,136 +78,141 @@ u32 gGsFlagsMasks[] = { 0x000000FF, 0x0000FF00, 0x00FF0000, 0xFF000000 };
+    gItemIconOddPotionTex,           // ITEM_ODD_POTION
+    gItemIconPoachersSawTex,         // ITEM_POACHERS_SAW
+    gItemIconBrokenBiggoronSwordTex, // ITEM_BROKEN_BGS
+    gItemIconPrescriptionTex,        // ITEM_PRESCRIPTION
+    gItemIconEyeballFrogTex,         // ITEM_EYEBALL_FROG
+    gItemIconEyeDropsTex,            // ITEM_EYE_DROPS
+    gItemIconClaimCheckTex,          // ITEM_CLAIM_CHECK
+    gItemIconBowFireTex,             // ITEM_BOW_ARROW_FIRE
+    gItemIconBowIceTex,              // ITEM_BOW_ARROW_ICE
+    gItemIconBowLightTex,            // ITEM_BOW_ARROW_LIGHT
+    gItemIconSwordKokiriTex,         // ITEM_SWORD_KOKIRI
+    gItemIconSwordMasterTex,         // ITEM_SWORD_MASTER
+    gItemIconSwordBiggoronTex,       // ITEM_SWORD_BIGGORON
+    gItemIconShieldDekuTex,          // ITEM_SHIELD_DEKU
+    gItemIconShieldHylianTex,        // ITEM_SHIELD_HYLIAN
+    gItemIconShieldMirrorTex,        // ITEM_SHIELD_MIRROR
+    gItemIconTunicKokiriTex,         // ITEM_TUNIC_KOKIRI
+    gItemIconTunicGoronTex,          // ITEM_TUNIC_GORON
+    gItemIconTunicZoraTex,           // ITEM_TUNIC_ZORA
+    gItemIconBootsKokiriTex,         // ITEM_BOOTS_KOKIRI
+    gItemIconBootsIronTex,           // ITEM_BOOTS_IRON
+    gItemIconBootsHoverTex,          // ITEM_BOOTS_HOVER
+    gItemIconBulletBag30Tex,         // ITEM_BULLET_BAG_30
+    gItemIconBulletBag40Tex,         // ITEM_BULLET_BAG_40
+    gItemIconBulletBag50Tex,         // ITEM_BULLET_BAG_50
+    gItemIconQuiver30Tex,            // ITEM_QUIVER_30
+    gItemIconQuiver40Tex,            // ITEM_QUIVER_40
+    gItemIconQuiver50Tex,            // ITEM_QUIVER_50
+    gItemIconBombBag20Tex,           // ITEM_BOMB_BAG_20
+    gItemIconBombBag30Tex,           // ITEM_BOMB_BAG_30
+    gItemIconBombBag40Tex,           // ITEM_BOMB_BAG_40
+    gItemIconGoronsBraceletTex,      // ITEM_STRENGTH_GORONS_BRACELET
+    gItemIconSilverGauntletsTex,     // ITEM_STRENGTH_SILVER_GAUNTLETS
+    gItemIconGoldenGauntletsTex,     // ITEM_STRENGTH_GOLD_GAUNTLETS
+    gItemIconScaleSilverTex,         // ITEM_SCALE_SILVER
+    gItemIconScaleGoldenTex,         // ITEM_SCALE_GOLDEN
+    gItemIconBrokenGiantsKnifeTex,   // ITEM_SWORD_KNIFE
+    gItemIconAdultsWalletTex,        // ITEM_ADULTS_WALLET
+    gItemIconGiantsWalletTex,        // ITEM_GIANTS_WALLET
+    gItemIconDekuSeedsTex,           // ITEM_DEKU_SEEDS
+    gItemIconFishingPoleTex,         // ITEM_FISHING_POLE
+    // icon_item_static 16x24 ia8
+    gSongNoteTex, // ITEM_SONG_MINUET
+    gSongNoteTex, // ITEM_SONG_BOLERO
+    gSongNoteTex, // ITEM_SONG_SERENADE
+    gSongNoteTex, // ITEM_SONG_REQUIEM
+    gSongNoteTex, // ITEM_SONG_NOCTURNE
+    gSongNoteTex, // ITEM_SONG_PRELUDE
+    gSongNoteTex, // ITEM_SONG_LULLABY
+    gSongNoteTex, // ITEM_SONG_EPONA
+    gSongNoteTex, // ITEM_SONG_SARIA
+    gSongNoteTex, // ITEM_SONG_SUN
+    gSongNoteTex, // ITEM_SONG_TIME
+    gSongNoteTex, // ITEM_SONG_STORMS
```

Comment:
Be nice if these could line up with others, although the formatter won't have it I'm sure.

Reply:
The others don't line up either, they just misalign less
idk I like it like this, it helps highlighting the "sections" imo